restapi.go

```diff
@@ -2162,11 +2162,20 @@ func (s *Session) WebhookDeleteWithToken(webhookID, token string) (st *Webhook,
 // webhookID: The ID of a webhook.
 // token    : The auth token for the webhook
 // wait     : Waits for server confirmation of message send and ensures that the return struct is populated (it is nil otherwise)
-func (s *Session) WebhookExecute(webhookID, token string, wait bool, data *WebhookParams) (st *Message, err error) {
+// threadID :	Sends a message to the specified thread within a webhook's channel. The thread will automatically be unarchived.
+func (s *Session) WebhookExecute(webhookID, token string, wait bool, threadID string, data *WebhookParams) (st *Message, err error) {
```

Comment:
I'm not sure that changing the signature of the established method is the right way here—adding another method might be a better solution. If we're forcing users to change their code, it should be for a necessary reason, not for adding `""` in the vast majority of cases.